internal/pkg/describe/webapp.go

```diff
@@ -332,6 +344,46 @@ func (d *WebAppDescriber) stackDescriber(roleARN, region string) (stackDescriber
 	return d.stackDescribers[roleARN], nil
 }
 
+// ServiceArn is the arn of an ECS service.
+type ServiceArn string
+
+// ClusterName returns the cluster name.
+// For example: arn:aws:ecs:us-west-2:1234567890:service/my-project-test-Cluster-9F7Y0RLP60R7/my-project-test-my-app-Service-JSOH5GYBFAIB
+// will return my-project-test-Cluster-9F7Y0RLP60R7
```

Comment:
hehe this is only safe after https://github.com/aws/amazon-ecs-cli-v2/pull/793, at the moment the ARN won't have the cluster name unless the root user opted in to long arns.

So I'd block merging this PR until #793 is in.

Reply:
Just unblocked your block :)

Reply:
Good to know "at the moment the ARN won't have the cluster name unless the root user opted in to long arns"!

Reply:
Just thinking out loud - you should also get the cluster from the `describeStackResources` right? So we could make this a backwards compatible change.

Reply:
That's true. But since the ARN resource path pattern for ECS service ({clusterName/serviceName}) is not likely to change, I think it is safe for us the get the cluster name by parsing the ARN?